src/components/FormElements/FormikSelect/FormikSelect.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from "react";
+
+import { createStyles, makeStyles, MenuItem } from "@material-ui/core";
+import FormControl from "@material-ui/core/FormControl";
+import FormHelperText from "@material-ui/core/FormHelperText";
+import InputLabel from "@material-ui/core/InputLabel";
+import { SelectProps } from "@material-ui/core/Select";
+import { Field, useFormikContext } from "formik";
+import { Select } from "formik-material-ui";
+
+type SelectOptionValue = string | number;
+
+type SelectOption =
+  | {
+      label: React.ReactNode;
+      value: SelectOptionValue;
+    }
+  | SelectOptionValue;
```

Comment:
same here as for the radio group, only have the `label` and `value` option to keep it simple